helm-chart/eoapi/values.yaml

```diff
@@ -39,11 +39,19 @@ service:
   port: 8080
 
 ingress:
-  # `"nginx"` will create a `kind:Service` with a `spec.port:ClusterIP`
-  # and a single Load Balancer and path rewrites for /vector, /stac, /raster
+  # Unified ingress configuration for both nginx and traefik
   enabled: true
+  # ingressClassName: "nginx" or "traefik"
   className: "nginx"
+  # Path configuration
+  pathType: "Prefix"  # Can be "Prefix" or "ImplementationSpecific" based on controller
+  pathSuffix: ""      # Add a suffix to service paths (e.g. "(/|$)(.*)" for nginx regex)
```

Comment:
We should consider adding a note here to users that they'll need to change the pathType to "ImplementationSpecific" if a regex is defined as pathSuffix for nginx. https://github.com/developmentseed/eoapi-k8s/pull/189 has more context.